### DISCUSSION TITLE
Can't figure out regexManagers to work with buildspec.yml (codebuild)

### DISCUSSION THREAD

**Discussion by qizhang1** [renovatebot/renovate#21638] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

Ronovate to discover a version below in buildspec.yml 
docker image is public image=bitnami/external-dns:0.13.3 notice, this is codebuild buildspec.yml file not dockerfile
env:
  variables:
    # renovate: image=bitnami/external-dns
    EXTERNAL_DNS_VERSION: 0.13.3

### What you tried so far.

regexManaers

    {
      "datasourceTemplate": "docker",
      "fileMatch": ["(^|/)buildspec\\.yml$"],
      "matchStrings": [
        "#\\s?renovate: image=(?<depName>.*?)\\s?EXTERNAL_DNS_VERSION:\\s?(?<currentValue>.*?)\\s"
      ]
    }

### Relevant debug logs

N/A no error but not discovered in the PR

**Comment by viceice** [comment]
https://regex101.com/r/ZigQ2D/1

you probably need a trailing newline in yourt `buildspec.yml`, otherwise it matches the space after `EXTERNAL_DNS_VERSION:` as current value
